### DISCUSSION TITLE
forum-session-Sonification

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#95] [General]
# forum-session-Sonification

https://accessibility2024.arxiv.org/forum-session-Sonification

**Comment by pauljgreen** [comment]
Please consider highlighting/discussing the Sensing the Dynamic Universe project
       https://lweb.cfa.harvard.edu/sdu
I'd be happy to give a brief presentation if you like.

Cheers
Paul

**Comment by SBBCornell** [comment]
Welcome everyone! Please share thoughts and resources of interest to the community.

**Comment by erichonour** [comment]
Looking forward to this! There is a great collection of data sonification examples here, if it's useful to anyone:

https://sonification.design

**Comment by car222222** [comment]
Another one in the middle of the night for some of us!  Sorry not to be there.

**Reply by SBBCornell** [reply]
We will be sharing recordings of all sessions afterwards. Once we add accessible captions we'll make those available on arXiv's youtube channel. I hope you can watch it then!

**Comment by baeolophus** [comment]
Will this session be one hour or 1.5 hours long?  Thanks!

**Reply by SBBCornell** [reply]
It will be 1.5 hours long, from 3-4:30 Eastern Time.

**Comment by engek** [comment]
This recently published State-of-the-Art Report on the integration of sonification and visualization might be of interest, even if it is not about accessibility: https://onlinelibrary.wiley.com/doi/full/10.1111/cgf.15114

**Comment by julianna-langston** [comment]
What tools do you use to produce or work with sonification?

What do you consider to be the biggest barriers to widespread adoption of sonification in science?

**Reply by erichonour** [reply]
Personally, I've used both Max (https://cycling74.com/) and Python to work with sonification. I don't know if it's the biggest barrier, but listening precisely is a skill that takes a while to develop, and feels pretty foreign to a lot of people, in my experience. I think really great sonification design can help with that, but it's still a barrier.

**Comment by SBBCornell** [comment]
We just wrapped the wonderful live session in which multiple attendees shared resources to additional sonification tools that I want to share here. Did I miss a link you shared? Please post it!

- Quick shout out to [Decibels](https://decibels.community/), a data sonification community with lots of folks talking about these topics.
- I'd just like to shout out my own sonification tool - [Chart2Music](https://github.com/julianna-langston/chart2music). It's javascript-based, and focuses on supporting interactivity (go forward, back, jump around, replay, speed up or slow down, etc). 
- Regarding python and sonification: I would like to point you all towards the work of Dennis Reinsch and Thomas Hermann: https://pub.uni-bielefeld.de/record/2979095, https://github.com/interactive-sonification. Three projects should be especially interesting: sc3nb, mesonic, and sonecules. They bridge the gap between python and SuperCollider (and other backends), a widely used sound synthesis software used in the sonification community.
- A third tactile display I am aware of as well as the Monarch and Graphiti is the [Canute](https://bristolbraille.org/about-canute-console-the-next-generation-in-braille/)
